src/model/exceptions/bad-request-exception.ts

```diff
@@ -0,0 +1,8 @@
+import { Exception } from './exception'
+import { ErrorCode } from '../enums/error-code'
+
+export class BadRequestException extends Exception {
```

Comment:
This exception isn't used anywhere.
Also, I think this is a bit too broad to be an exception in our system. We want our exceptions to be descriptive of the error that happend, so we (almost immediatly) have an idea of what is wrong, just by the exception name.
Since this exception isn't used, I suggest to delete it again.

Reply:
400 Bad Request is a very common HTTP error code in a RESTful API. I suggest we use it for future use in API controllers.

Reply:
Yes, it is a HTTP error code, but our Exceptions are not exceptions describing HTTP, they are exceptions describing things going wrong in Sofie.
We _then_ map our Exceptions to HTTP statusses. You will find in ExpressErrorHandler.ts that we map our Exceptions to HTTP statusses:
![image](https://github.com/user-attachments/assets/095a3de9-1764-4f3f-b1db-37e0d6690c68)

Reply:
Okay. I can work with that. A lot of .NET code houses actually bind up HttpStatuscodes with exceptions to get the stack trace (which is a malpractice in regards to security), but if this is the case I can work with this format instead:

response.status(400).send('Bad Request')

Reply:
Turns out express handles malformed json for us...

Reply:
That sounds good. Just remember to still delete this class `BadRequestException` :)